### PR TITLE
Add whitelist guild names and tags option.

### PR DESCRIPTION
**Pull request by Rollczi**

### CODE REVIEW

File: src/main/java/net/dzikoysk/funnyguilds/data/configs/PluginConfiguration.java [hunk 1]
```diff
@@ -1100,13 +1100,19 @@
...
-    @CustomKey("restricted-guild-names")
-    public List<String> restrictedGuildNames = Collections.singletonList("Administracja");
+    @CustomKey("blacklist-guild-names")
+    public List<String> blacklistGuildNames = Collections.singletonList("Administracja");
 
     @Comment("Niedozwolone tagi przy zakladaniu gildii")
-    @CustomKey("restricted-guild-tags")
-    public List<String> restrictedGuildTags = Collections.singletonList("TEST");
+    @CustomKey("blacklist-guild-tags")
+    public List<String> blacklistGuildTags = Collections.singletonList("TEST");
```
Comment (PinkLolicorn) [inline src/main/java/net/dzikoysk/funnyguilds/data/configs/PluginConfiguration.java:1115]: Please do not change config keys when there is no real need for it.

File: src/main/java/net/dzikoysk/funnyguilds/data/configs/PluginConfiguration.java [hunk 2]
```diff
@@ -1100,13 +1100,19 @@
     @CustomKey("check-for-restricted-guild-names")
     public boolean checkForRestrictedGuildNames = false;
```
Comment (PinkLolicorn) [inline src/main/java/net/dzikoysk/funnyguilds/data/configs/PluginConfiguration.java:1101]: Do not make such unrelated changes.

File: src/main/java/net/dzikoysk/funnyguilds/data/configs/PluginConfiguration.java [hunk 3]
```diff
@@ -1201,18 +1207,18 @@
...
     @Min(0)
-    @Comment("Margines sprawdzania przez ile bloków uderzył gracz w serce gildii")
+    @Comment("Margines sprawdzania przez ile blokow uderzyl gracz w serce gildii")
     @CustomKey("freeCam-compensation")
     public int freeCamCompensation = 0;
 
     @Min(1)
-    @Comment("Ilość wątków używanych przez ConcurrencyManager")
+    @Comment("Ilosc watkow uzywanych przez ConcurrencyManager")
     @CustomKey("concurrency-threads")
```
Comment (PinkLolicorn) [inline src/main/java/net/dzikoysk/funnyguilds/data/configs/PluginConfiguration.java:1222]: Do not make such unrelated changes.

**Review by PinkLolicorn — CHANGES_REQUESTED** [review]

File: src/main/java/net/dzikoysk/funnyguilds/data/configs/PluginConfiguration.java (hunk 1 above)
Comment (P3ridot) [reply src/main/java/net/dzikoysk/funnyguilds/data/configs/PluginConfiguration.java:1115]: When whitelist = true, name *blacklist* make less sens than restricted so why did you change that?
